src/components/Tables/Table.tsx

```diff
@@ -134,32 +107,57 @@ const defaultColumnStyleOptions: ColumnStyleOptions = {
     maxWidth: "20rem",
 };
 
-const Table: React.FC<Props> = ({
+const defaultSortFunction = <T extends unknown>(left: T, right: T): number => {
```

Comment:
why isn't this just default??? is this a 'javascript bad' moment?

Reply:
Nope this is an "I'm bad" moment